composer.json

```diff
@@ -12,7 +12,10 @@
     },
     "require-dev": {
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+        "doctrine/rst-parser": "dev-master",
         "phpcompatibility/php-compatibility": "^9.3",
+        "phpdocumentor/phpdocumentor": "dev-master",
```

Comment:
This (and it's friends) need to be the `dev-master` version for now because phpDocumentor's official release does not support some 7.x features and will not generate documentation for classes that use them otherwise